test/cppsuite/configs/background_compact_long.txt

```diff
@@ -0,0 +1,55 @@
+# Configuration for background_compact.
+# ~2 hour run time.
+duration_seconds=7200,
+cache_size_mb=10000,
+timestamp_manager=
```

Comment:
ditto re. the `timestmap_manager`

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9691/commits/0014dfa45ae7c27ab3a958f59f8131232b05ea77